internal/host/host.go

```diff
@@ -86,6 +86,7 @@ type Config struct {
 	ResetTimeout            time.Duration           `envconfig:"RESET_CLUSTER_TIMEOUT" default:"3m"`
 	MonitorBatchSize        int                     `envconfig:"HOST_MONITOR_BATCH_SIZE" default:"100"`
 	DisabledHostvalidations DisabledHostValidations `envconfig:"DISABLED_HOST_VALIDATIONS" default:""` // Which host validations to disable (should not run in preprocess)
+	BootstrapHostMAC        string                  `envconfig:"BOOTSTRAP_HOST_MAC" default:""`
```

Comment:
Lets add a comment here that this param should be used only in case of a single cluster deployment. Because in podman it won't really work